### DISCUSSION TITLE
Place projects into folders and reorder files

### DISCUSSION THREAD

**Discussion by EmperorKabir** [pybricks/support#1385] [Ideas]
Currently on the left hand side, we just have a long list of project names and (as far as I can tell, please correct me if I'm wrong), we cannot reorder the files nor can we put them in folders/subdirectories. For obvious reasons, this would be really useful! I'd personally love a folder for my 'finished product' templates, another for works in progress, another for miscellaneous or when I show friends and family etc, another for where I have duplicated block code projects into code projects so I can work on the code etc etc etc.

**Comment by laurensvalk** [comment]
While not a perfect solution, you could prefix your project names as `finished_`, `wip_` for work in progress, and so on.

Since they are sorted alphabetically, you get some organization that way.

**Reply by BertLindeman** [reply]
It does make the program name in the tab just a tad more cryptic ;-)
So you have to depend on the pop-up on hover over the tab.
Or Just use one tabs at a time, so you know what program you work on.

Other features have a higher priority, understood.

**Reply by EmperorKabir** [reply]
Yeah and I don't think we can resize the column with the file names. I'll probably take your suggestion on file names but ideally, reordering and sub directories would be useful.
